lib/shopify_app/scripttags_manager.rb

```diff
@@ -3,10 +3,12 @@ class ScripttagsManager
     class CreationFailed < StandardError; end
 
     def self.queue(shop_domain, shop_token, scripttags)
-      ShopifyApp::ScripttagsManagerJob.perform_later(
-        shop_domain: shop_domain,
-        shop_token: shop_token,
-        scripttags: scripttags
+      ShopifyApp::ScripttagsManagerJob
+        .set(queue: ShopifyApp.configuration.webhooks_manager_queue_name)
```

Comment:
this is to allow dynamic config right? I would be okay with not having this part of the config be dynamic. I think the risk of someone being surprised by this would be low as in general I doubt many will use this setting.